css/typography.css

```
@@ -62,15 +62,21 @@ h1, .h1 {
 }
 
 h2, .h2 {
-    font-size: 3.2rem;
+    font-size: 2.8rem;
     font-weight: var(--font-weight-extrabold);
     line-height: 1.2;
+    margin: 1.5em 0 0.5em;
+}
+
+h2:first-of-type, .h2:first-of-type {
+    margin: 0.75em 0 0.5em;
```

```suggestion
    margin-top: 0.75em;
```
It's explicitly only the top margin that's changing here, right?